crates/next-transform-dynamic/tests/fixture.rs

```diff
@@ -0,0 +1,94 @@
+use std::path::{Path, PathBuf};
+
+use next_transform_dynamic::{next_dynamic, NextDynamicMode};
+use swc_core::{
+    common::FileName,
+    ecma::{
+        parser::{EsConfig, Syntax},
+        transforms::testing::{test, test_fixture},
+    },
+};
+use testing::fixture;
+
+fn syntax() -> Syntax {
+    Syntax::Es(EsConfig {
+        jsx: true,
+        ..Default::default()
+    })
+}
+
+#[fixture("tests/webpack/fixture/**/input.js")]
+fn next_dynamic_webpack_fixture(input: PathBuf) {
+    next_dynamic_fixture(
+        &input,
+        "output-dev.js",
+        true,
+        false,
+        false,
+        NextDynamicMode::Webpack,
+    );
+    next_dynamic_fixture(
+        &input,
+        "output-prod.js",
+        false,
+        false,
+        false,
+        NextDynamicMode::Webpack,
+    );
+    next_dynamic_fixture(
+        &input,
+        "output-server.js",
+        false,
+        true,
+        false,
+        NextDynamicMode::Webpack,
+    );
+}
+
+#[fixture("tests/turbo/fixture/**/input.js")]
+fn next_dynamic_turbo_fixture(input: PathBuf) {
+    // TODO(alexkirsz) Also test production once implemented.
+    next_dynamic_fixture(
+        &input,
+        "output-dev-client.js",
+        true,
+        false,
+        false,
+        NextDynamicMode::Turbo,
+    );
+    next_dynamic_fixture(
+        &input,
+        "output-dev-server.js",
+        true,
+        true,
+        false,
+        NextDynamicMode::Turbo,
+    );
+}
```

Comment:
Instead of duplicating the input files you could run these on the original cases with a different output file prefix:

``` rs
    next_dynamic_fixture(
        &input,
        "output-turbo-dev-server.js",
        true,
        true,
        false,
        NextDynamicMode::Turbo,
    );
```